combat-trainer.lic

```diff
@@ -2088,6 +2088,15 @@ class AttackProcess
     @ambush_location = settings.ambush_location
     echo("  @ambush_location: #{@ambush_location}") if $debug_mode_ct
 
+    @get_actions = %w[get wield]
+    echo("  @get_actions: #{@get_actions}") if $debug_mode_ct
```

Comment:
💡 I'm not sure if there's much value in echoing something that isn't variable. On the other hand, it's not that much extra logging.

Reply:
Thought this too, but it could be helpful with debugging what is expected vs what we put in. It's only output once, so I figured it would be ok.